src/React.Core/Babel.cs

```diff
@@ -79,6 +79,53 @@ public Babel(IReactEnvironment environment, ICache cache, IFileSystem fileSystem
 			_babelConfig = siteConfig.BabelConfig.Serialize();
 		}
 
+        /// <summary>
+        /// Transforms a JavaScript literal. Results of the transformation are cached.
+        /// </summary>
+        /// <param name="scriptId">The id of the script, used for caching purposes</param>
+        /// <param name="script">The script contents</param>
+        /// <returns>JavaScript</returns>
+        public virtual string TransformLiteral(string scriptId, string script)
```

Comment:
This name is pretty confusing, what about calling it `TransformWithSourceMapCached` or something like that?

Also, please add unit tests for this method.